src/main/java/org/jasig/portlet/newsreader/adapter/INewsAdapter.java

```diff
@@ -47,6 +48,6 @@
      */
     String getDescriptionKey();
 
-    NewsFeed getSyndFeed(NewsConfiguration config, PortletRequest request) throws NewsException;
+    NewsFeed getSyndFeed(NewsConfiguration config, int page) throws NewsException;
```

Comment:
There's something a little funny about this change.  The need for the 'page' argument is clear, but removing the request argument doesn't add up.  Now that we're doing so much more with non-portlet requests inside out heavy JS portlet UIs, we ARE trying to make our service APIs less tied to the Portlet API (so we can access them within an HttpServletRequest).  But in this case, the solution in the PR just turns around and calls RequestContextHolder.currentRequestAttributes()... and if the current request isn't a PortletRequest, it throws an exception.